.github/workflows/prepare_release.yml

```
@@ -0,0 +1,67 @@
+---
+
+name: Prepare Release
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: 'Module version to be released.'
+        required: true
```

I would love it if we had some logic to automatically detect this based on labels, making it optional but when I looked at it I had trouble coming up with a good solution so this is a good first step.

---

dito. I would also love to do this automatically. but one step at a time.

---

After https://github.com/voxpupuli/gha-puppet/pull/66#discussion_r1894705081 this can be
```suggestion
        required: false
```